packages/trader/src/Stores/Modules/Trading/Actions/start-date.ts

```diff
@@ -58,7 +58,7 @@ export const onChangeStartDate: TOnChangeStartDate = async store => {
 
 export const onChangeExpiry: TOnChangeExpiry = async store => {
     const { start_time, expiry_date, expiry_type, expiry_time, start_date, symbol, sessions } = store;
-
+    // @ts-expect-error TODO: check if TS error is gone after ContractType is migrated to TS
     const trading_times = await ContractType.getTradingTimes(expiry_date, symbol);
```

Comment:
Btw it was migrated to TS in package 2
https://github.com/binary-com/deriv-app/pull/9284/files#diff-8a78d4d4d6bae57aa44dc11fbb9091d409f34a80c36407204e3151e7435ad72d
I will check this //TODO after 2d package will be merged